tastyworks/models/alert.py

```diff
@@ -0,0 +1,46 @@
+from decimal import Decimal
+from enum import Enum
+
+from dataclasses import dataclass
+
+
+class AlertField(Enum):
+    LAST = 'Last'
+    BID = 'Bid'
+    ASK = 'Ask'
+    IVX = 'IV'
+
+
+class Operator(Enum):
+    LESSTHAN = '<'
+    GREATERTHAN = '>'
+
+
+@dataclass
+class Alert:
+    alert_field: AlertField
+    operator: Operator
+    symbol: str
+    threshold: Decimal
+    alert_external_id: str = ''
+    user_external_id: str = ''
+
+    def get_json(self):
+        alert_json = {'field': self.alert_field.value,
+                      'operator': self.operator.value,
+                      'threshold': '{:.3f}'.format(self.threshold),
+                      'symbol': self.symbol
+                      }
+        return alert_json
+
+    @staticmethod
+    def from_dict(data: dict):
```

Comment:
please use `typing.Dict`. e.g. `from typing import Dict` and then set the hint to `Dict` for this argument

Reply:
Also would be great if you then hinted the return type to be a list of `Alert`, e.g. `def from_dict(data: Dict) -> List[Alert]:`, the `List` hint is also in the `typing` class